open_rfps/SubSquare_integration_proposal.md

```diff
@@ -0,0 +1,71 @@
+## Due Date
+May 2022
+
+## Overview
+[SubSquare](https://www.subsquare.io/) is a governance platform designed for substrate based blockchains. It scans the 
+governance, normalize the on-chain data and provide interfaces for community users to go ahead with the governance
+workflow by discussion, voting, proposing, etc. We propose to integrate SORA solo chain which we think will greatly help 
+its governance workflow.
+
+The advantage of subsquare compared to polkassembly and commonwealth include:
+- More accurate on chain events monitor and extraction.
+- More detailed timeline.
+- Fast team response time.
+- All in all, the UI/UX is better.
+- We have plans to make it a web3 apps and decentralized, but it will be long termed work.
+- OpenSquare other products support in high priority, including off-chai voting, bounties, paid QA, etc.
+
+Please check features catalog doc [here](https://opensquare.feishu.cn/docs/doccnv8EGKC5QtegMnmHnNZcK5f) which includes 
+the basic features and advantage of subsquare.
+
+## Proposal goals
+Enhance SORA solo chain's governance visualization and workflow.
+
+## Scope of work
+- Scan SORA solo chain governance data and save the normalized data to database.  
+- SORA related fronted design and backend configuration.
+- A restful server which will serve SORA identities which will be used by subsquare to show the identity.
+- Deploy SORA subsquare implementation.
+
+The basic features of this implementation will include:
+
+- Democracy proposals/referendums visualization and discussion.
+- Council/Technical committee motion visualization and discussion.
+- Overview and discussions feature.
+
+## Current roadblocks and barriers to success
+None.
```

Comment:
Please include roadblocks and barriers to success here, they can be as simple as this implementation not existing yet, or as complex as meriting a detailed breakdown.

Reply:
Thanks, added ddfa8d067463b52fbc0f9a2edfebf287eabd405c. Tell me if it's still not enough.